src/System.Management.Automation/utils/CryptoUtils.cs

```diff
@@ -329,26 +329,13 @@ public PSCryptoException(string message, Exception innerException)
         /// <param name="context">Context in which this constructor is called.</param>
         /// <remarks>Currently no custom type-specific serialization logic is
         /// implemented</remarks>
+        [Obsolete("Legacy serialization support is deprecated since .NET 8", DiagnosticId = "SYSLIB0051")] 
```

Comment:
`[Serializable]` attribute was not removed from this class.

Reply:
removed